static/src/javascripts/projects/commercial/modules/prebid/prebid.js

```diff
@@ -28,6 +28,22 @@ const consentManagement = {
     allowAuctionWithoutConsent: true,
 };
 
+const s2sConfig = {
+    accountId: '1',
+    enabled: true,
+    bidders: [
+        'appnexus',
+        'openx', // Defined in the doc.
+    ],
+    timeout: 1000,
```

Comment:
Why does the timeout need to be shorter than the auction timeout?

Reply:
Don't know. I will change it to 1500 :)

Reply:
If it can inherit the same value as the auction that would be great.